### ISSUE TITLE
Strange entries in rdb$Relation_Fields.rdb$Default_Source 

### ISSUE THREAD

**Issue by Schnaddel** [FirebirdSQL/firebird#7210]
We have a ddl statement as follows:

```sql
create table TCLF_ChangeLogField
( CLF_Nr        integer      generated by default as identity not null
, CLF_CLT_Nr    integer      default 0 /* via Trigger */      not null
, CLF_UT_Nr     integer      default 0 /* via Trigger */      not null
);
```

In the past the field rdb$Default_Source in the system table rdb$Relation_Field contained the value "default 0".
After update to 3.0.10 it contains "default 0 /* via Trigger */".

This raises serous problems. Please investigate and fix.

**Comment by mrotteveel** [comment]
As to the automatic removal of the asterisk, that is because you entered the code in "plain" text, instead of in a code block (enclosed in \`\`\`), and `*` in markdown marks the start/end of italics. You can keep them in plain text by escaping them (i.e. `\*`), but instead I have edited it to make it into a code block.

**Comment by mrotteveel** [comment]
What kind of serious problems does it raise?

**Comment by mrotteveel** [comment]
As far as I'm aware, it is only used to reconstruct the DDL source when using `isql -ex` (and similar by other tools that reconstruct DDL source). The actual default is stored in compiled BLR form in `RDB$DEFAULT_VALUE`.

**Comment by aafemt** [comment]
It was intended change as part of fix for #6698 and similar issues.

**Comment by Schnaddel** [comment]
Regarding the fix #6698: I noticed (and wondered) that after updating to 3.0.10, several stored procs were considered changed during our build process. All of them had comments like this:
...
RETURNS (RSTATUS INTEGER)
AS
/* RStatus-Values:
    2=OK (updated)
    1=OK (inserted)
    0=nothing to do
   -5=check failed
   -7=foreign key invalid
*/
  DECLARE VARIABLE LNr integer;
...

So from that perspective, keeping the comments is an improvement. If the DDL is regenerated from the content of the Default_Source field, it can even be restored including the comment. 

I will change our build process to properly handle the comments in the Default_Source field.
